main.py

```diff
@@ -1,26 +1,23 @@
-import sys
 from inputvalidations import Validator
-from os import path
-from reader import Reader
-from parser import Parser
-from calculator import Calculator
-from utils import print_most_humid_day, print_highest_temperature, print_lowest_temperature
+from controller import Controller
+from utils import read_file_path, read_year, read_switch, is_digit, typcast_year
 
 if __name__ == '__main__':
-    file_path = sys.argv[1]
-    year = int(sys.argv[3])
     if (
-            sys.argv[2] == '-e' and path.isdir(file_path) and Validator.year_validation(year, file_path)
+            Validator.is_argument_valid() and read_switch() == '-e' and Validator.is_dir()
+            and Validator.is_year_valid()
```

Comment:
can we move these three conditions in one codition and use that one condition here instead?